run/localizer/constants.ts

```diff
@@ -0,0 +1,7 @@
+export enum LOCALE_DEFAULTS {
```

Comment:
@burtonemily Just a note this will definitely need to be updated when the network page goes into QA